### PR TITLE
Entries: Document assert#pushResult and deprecate assert#push

### LINKED ISSUE #120: Document Assert#pushResult
Document Assert.prototype.pushResult + deprecated Assert#push

### PR DESCRIPTION
**Pull request by Arkni**
Fixes #120

### CODE REVIEW

**Comment by Arkni** [comment]
@leobalter 
Just checked the api documentation and the properties of the `assertionResult` object are not shown.
screenshot from 2016-03-25 21 21 49

Is this have something to do with the markup ?

File: entries/pushResult.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="pushResult">
+    <title>pushResult()</title>
+    <signature>
+        <argument name="assertionResult" type="PlainObject">
+            <desc>The assertion result</desc>
+            <propery name="result" type="Boolean">
```
Comment (trentmwillis) [inline entries/pushResult.xml]: Thinking this should probably be "property", missing the "t"

Comment (Arkni) [inline entries/pushResult.xml]: Opss, my bad.
I'll open another proposal to fix that.
Sorry about the mess :(

Comment (Arkni) [inline entries/pushResult.xml]: Opened #125

**Pull request closed without merging**